examples/validate-fee/project/Dependencies.scala

```diff
@@ -0,0 +1,52 @@
+import sbt.*
+
+object Dependencies {
+
+  object V {
+    val tessellation = "2.9.1"
+    val decline = "2.4.1"
+    val http4s = "0.23.16"
+  }
+
+  def tessellation(artifact: String): ModuleID = "org.constellation" %% s"tessellation-$artifact" % V.tessellation
+
+  def http4s(artifact: String): ModuleID = "org.http4s" %% s"http4s-$artifact" % V.http4s
+
+  def decline(artifact: String = ""): ModuleID =
+    "com.monovore" %% {
+      if (artifact.isEmpty) "decline" else s"decline-$artifact"
+    } % V.decline
+
+  object Libraries {
+    val tessellationNodeShared = tessellation("node-shared")
+    val tessellationCurrencyL0 = tessellation("currency-l0")
+    val tessellationCurrencyL1 = tessellation("currency-l1")
```

Comment:
if possible, the only dependency to tessellation should be the `sdk`

Reply:
@marcinwadon I'll look into this more later but replacing these with "sdk" causes the euclid build to fail. It's not able to download/see the sdk jar, which I do see in my `.ivy/` directory.

Reply:
Sure. I think it may be the euclid that does not assembly the sdk module but only those you used.

Reply:
That is correct. Currently Euclid assembles and publishes[ these packages](https://github.com/Constellation-Labs/euclid-development-environment/blob/bc8f08e4b7b90161171b325e1e524c7c6f86c146/infra/docker/metagraph-ubuntu/Dockerfile#L35)
```
 sbt kernel/publishM2 shared/publishM2 keytool/publishM2 nodeShared/publishM2 dagL1/publishM2 currencyL0/publishM2 currencyL1/publishM2;
 
 sbt wallet/assembly dagL0/assembly dagL1/assembly keytool/assembly;
 ```

Reply:
Looking at recent releases on Tessellation, I'm not sure if the [PR](https://github.com/Constellation-Labs/tessellation/pull/912) modifying  the `sdk` modules has been merged into a release yet.